services/m3dbnode/config/config_test.go

```diff
@@ -143,6 +143,7 @@ pooling:
         size: 524288
         lowWatermark: 0.01
         highWatermark: 0.02
+        maxFinalizerCapacity: 4
```

Comment:
nit: could you change it to the non-default value

And add some a test to ensure we use the default value when nothing is set